src/reducers/profileReducer.ts

```diff
@@ -1,21 +1,56 @@
 import { Action } from "../actions";
 import { ProfileActions } from "../actions/profileActions";
-import { Profile, UserMode } from "../state/profile";
+import { ApartmentType, GenderType, ProfileProps, UserRole } from "../state/profile";
 
-const defaultState: Profile = {
+const defaultState: ProfileProps = {
   basicDetails: "",
-  mode: UserMode.Tenant,
+  bio: {
+    age: 27,
+    education: "Masters",
+    email: "hello.world@tum.de",
+    fname: "Shoaib",
+    gender: GenderType.Male,
+    language: "English, German",
+    lname: "Khan",
+    phoneNumber: "1232312134",
+  },
+  description: "hello I am searching for an accommodation",
+  editMode: false,
+  realState: [
+    {
+      UTMCords: [123213, 1231],
+      address: "Friesenger Landstraße 47A Garching b. München 85748",
+      area: 20,
+      description: "A bright room in a furnished three-room apartment. You will be sharing the bathroom and the kitchen with two other tenants.",
+      distanceToUni: 2,
+      key: "property1",
+      name: "Room a in a shared apartment",
+      onSiteParking: true,
+      onSiteStorage: false,
+      petFriendly: true,
+      type: ApartmentType.WG,
+    },
+  ],
+  role: UserRole.Tenant,
+  specs: {
+    budget: "100-500",
+    leaseLength: 5,
+    onsiteParking: true,
+    onsiteStorage: true,
+    petFriendly: false,
+    searchingFor: ApartmentType.House,
+    smokeFriendly: false,
+  },
```

Comment:
The default state should be something that's default for everybody. Please extract this test state to the test, and provide a more generic default state here.

Reply:
i did not understand

Reply:
plz i will after the development plz plz T_T

Reply:
Fine...